src/styles/globals.css

```diff
@@ -2,6 +2,10 @@
 @tailwind components;
 @tailwind utilities;
 
+:root {
+  --nextui-notAnalyzed: 0, 0%, 55%;
```

Comment:
do we need this if it is already defined in tailwind config?

Reply:
the color was kinda transparent, but anyways I have applied another color and removed this part.